### PR TITLE
[excel] (Power Automate) ISO strict Open XML note

### PR DESCRIPTION
**Pull request by AlexJerabek**
This PR updates the PA troubleshooting article to mention strict Open XML workbooks aren't supported in flows.

Quick note to the writer reviewer, I used the casing found in [this article](https://learn.microsoft.com/en-us/DeployOffice/compat/office-file-format-reference). Happy to change if you have a better source.

### CODE REVIEW

**Comment by AlexJerabek** [comment]
Tagging @yutao-huang for a review (I can't add you as a formal reviewer).

**Comment by yutao-huang** [comment]
Do you think we should briefly explain or maybe add a reference to some official doc about ISO strict Open XML format? Or maybe we can assume if the user is currently using this format, they probably must have known what it is...?

**Review by yutao-huang — APPROVED** [review]

**Comment by AlexJerabek** [comment]
> Do you think we should briefly explain or maybe add a reference to some official doc about ISO strict Open XML format? Or maybe we can assume if the user is currently using this format, they probably must have known what it is...?

Sure! I couldn't find anything in MSFT docs, but I think this Library of Congress page should suffice: https://www.loc.gov/preservation/digital/formats/fdd/fdd000401.shtml

**Review by yutao-huang — APPROVED** [review]

File: docs/testing/power-automate-troubleshooting.md [hunk 1]
```diff
@@ -85,6 +85,10 @@ There are two reasons that the parameters or returned data of a script are not a
 
 The signature of a script is stored with the **Excel Business (Online)** connector when it is created. Remove the old connector and create a new one to get the latest parameters and return values for the **Run script** action.
 
+## ISO strict Open XML workbooks aren't supported
+
+The **Excel Business (Online)** connector's **Run script** action doesn't support workbooks with [ISO strict version of the Excel Workbook file format](https://www.loc.gov/preservation/digital/formats/fdd/fdd000401.shtml). Flows with this type of workbook return a "BadGateway" error when trying to run a script. This is due to coauthoring restrictions. Please save workbooks as the standard Excel Workbook format for use with Power Automate.
```
Comment (alison-mk) [inline docs/testing/power-automate-troubleshooting.md:90]: One tweak, for your consideration 
```suggestion
The **Excel Business (Online)** connector's **Run script** action doesn't support workbooks with the [ISO strict version of the Excel Workbook file format](https://www.loc.gov/preservation/digital/formats/fdd/fdd000401.shtml). Flows with this type of workbook return a "BadGateway" error when trying to run a script. This is due to coauthoring restrictions. Please save workbooks as the standard Excel Workbook format for use with Power Automate.
```

**Review by alison-mk — APPROVED** [review]
Looks good! Left one suggestion. Thanks for the context.

**Pull request merged**